docs/news.rst

```diff
@@ -13,6 +13,8 @@ Release Notes
 - Fixed ``ValueError: ZIP does not support timestamps before 1980`` when using
   ``SOURCE_DATE_EPOCH=0`` or when on-disk timestamps are earlier than 1980-01-01. Such
   timestamps are now changed to the minimum value before packaging.
+- The future-proof fix in 0.36.0 for converting PyPy's SOABI into a abi tag was
```

Comment:
I missed this: this line doesn't belong under v0.38.0 since it was already released before.

Reply:
right, sorry. Is there a place for the snippet to go? Should I add an `**UNRELEASED**` section?

Reply:
No worries, I'll handle that.